core/src/main.cpp

```diff
@@ -53,6 +53,12 @@
 #include <utility>
 #include <vector>
 
+#ifdef __cpp_lib_filesystem
+#  include <filesystem>
+#else
+#  include <boost/filesystem.hpp>
+#endif
```

Comment:
Just wondering: What situation would not have `std::filesystem` available for this project?

Reply:
Curious about this too. If we do need this, should this be done in the user mapping plugin as well for `local_file.cpp`?

Reply:
I think this is actually unnecessary now.

Confirmed via compiler explorer. Our compiler is new enough to not need this.

Will update.

Reply:
Done.